core/common/src/main/java/alluxio/conf/Hash.java

```diff
@@ -76,4 +78,11 @@ public String get() {
     }
     return mVersion;
   }
+
+  /**
+   * @return the latest update time
+   */
+  public long getLastUpdateTime() {
+    return mLastUpdateTime;
```

Comment:
need to consider update logics here

Reply:
add synchronized

Reply:
or can call get() and then return last update time since it doesn't need to be really accurate.
the get() will not need to sync when it check mShouldUpdate is false.
This way the getLastUpdateTime() does not need to block the actual update